otx/algorithms/segmentation/adapters/mmseg/models/segmentors/detcon.py

```diff
@@ -517,17 +516,17 @@ def __init__(
     # pylint: disable=arguments-renamed
     def forward_train(
         self,
-        img: torch.Tensor,
-        img_metas: List[Dict],
-        gt_semantic_seg: torch.Tensor,
-        pixel_weights: Optional[torch.Tensor] = None,
+        img,
```

Comment:
Is there a reason to clear the type hint?

Reply:
Please see here https://github.com/openvinotoolkit/training_extensions/pull/1911#discussion_r1142829506